content/en/docs/helm/helm_init.md

```diff
@@ -0,0 +1,9 @@
+---
+section: deprecated
```

Comment:
does this need a title as well?

```
title: "Helm Init"
```

I recall there being a bug raised about `helm upgrade` not showing up in the docs because of the missing title.

Reply:
https://github.com/helm/helm-www/issues/644

Reply:
thinking about this more... I guess we don't want this to actually show up in the nav. Just have this as a placeholder for those stumbling onto `helm init`.